lib/features/navigator/utils/navigation_commands.dart

```diff
@@ -44,4 +44,9 @@ extension NavigationX on WidgetRef {
   Future<void> navigateAboutUs() async {
     await _router.push(const AboutUsRoute());
   }
+
+  // TODO(mikolaj-jalocha): add method parameter to navigate to guide detail.
+  Future<void> navigateGuideDetail() async {
+    await _router.push(GuideDetailRoute(id: "1"));
```

Comment:
I guess i would already include this here as parameter and placed "1" in placeholder's invocation (in `GestureDetector` placeholder tile) . But it's obviously not that important, but we'll have one less place to change.

Reply:
Very good idea!